guides/release/tutorial/01-orientation.md

```diff
@@ -60,10 +60,10 @@ Could not start watchman
 Visit https://ember-cli.com/user-guide/#watchman for more info.
 building... 
 
-Build successful (28313ms) – Serving on http://localhost:4200/
+Build successful (9761ms) – Serving on http://localhost:4200/
```

Comment:
The variability here is a tad annoying (AFAICT nothing can be done about it, just mention the churn is a bit annoying)

Reply:
https://github.com/ember-learn/super-rentals-tutorial/pull/37 😉